code/src/temperature_frame_action.cpp

```diff
@@ -0,0 +1,21 @@
+#include <temperature_frame_action.hpp>
+
+namespace r2d2::robos {
+    temperature_frame_action_c::temperature_frame_action_c(
+        base_comm_c &comm, frame_s &frame, display_ostream_c &os)
+        : frame_action_c(comm, frame), ostream(os) {
+    }
+
+    void temperature_frame_action_c::reply_to_data() {
+        if (!changed) {
+            return;
+        }
+        // hwlib::cout << "Ambient temp: " << ambient_temperature / 10
```

Comment:
Please remove commented blocks of code before merging into master.

Reply:
fixed in https://github.com/R2D2-2019/RobOS/pull/3/commits/cfaff485e9ac1cc3207c3db54e5d30bd6be03395